plugins/dev-tools/package.json

```diff
@@ -38,7 +38,9 @@
   ],
   "dependencies": {
     "@types/dat.gui": "^0.7.5",
-    "dat.gui": "^0.7.7"
+    "chai": "^4.2.0",
+    "dat.gui": "^0.7.7",
+    "mocha": "^7.1.0"
```

Comment:
Mocha is already a dependency of dev-scripts, so I don't see why it needs to be a dependency here.